README.md

```diff
@@ -66,7 +66,9 @@ If the config was set up correctly, at this point you should see the daemon
 running smoothly.
 
 An additional env variable `LOG_LEVEL` can be set to regulate which logs will be
-shown. Its value can be one of `trace`, `debug`, `info`, `warn` or `error`.
+shown. Its value can be one of `trace`, `debug`, `info`, `warn` or `error`. It's
+also possible to hide logs from `ethers` by setting `RUST_LOG` to
+`none,defillama-answerer=info`
```

Comment:
Is this expected to be `none,defillama-answerer=info`? Not just `none`?  (also missing a dot I think).

Reply:
`none` would disable all logs from every crate. What I'm interested in is silencing `ethers` logs so that nothing would be logged if a connection is dropped and a reconnection happens automatically. That would only be noise.